### PR TITLE
adds submission import and removes command line calls

### PR DESCRIPTION
**Pull request by matthewpeterkort**
Don't think the try, except block over everything is a good way to catch and handle errors.
Tested and working with: quay.io/ohsu-comp-bio/aced-etl:latest-test
Is noticeably faster than the sub process command line calls.

### CODE REVIEW

File: etl-job/fhir_import_export.py [hunk 1]
```diff
@@ -241,7 +312,7 @@ def _main():
         raise Exception(f"unknown method {method}")
 
     # note, only the last output (a line in stdout with `[out]` prefix) is returned to the caller
-    print(f"[out] {json.dumps(output, separators=(',', ':'))}")
+    # print(f"[out] {json.dumps(output, separators=(',', ':'))}")
```
Comment (bwalsh) [inline etl-job/fhir_import_export.py:315]: gen3_util jobs reads this,  I think it would be better to continue to return it to the client

**Review by bwalsh — CHANGES_REQUESTED** [review]
Matthew,
Looks great.  Thank you for doing it, it is much faster.
I have one comment, to include the output json.

**Comment by bwalsh** [comment]
Also, we should delete the [load_all](https://github.com/ACED-IDP/aced_etl_pod/blob/main/etl-job/load_all) script

File: etl-job/fhir_import_export.py (hunk 1 above)
Comment (matthewpeterkort) [reply etl-job/fhir_import_export.py:315]: this line was causing a python error. Will investigate this further on Monday.

**Pull request merged**